docs/user-guide/troubleshooting/general.md

```diff
@@ -47,13 +47,26 @@ $ leverage tf shell
 [17:06:13.754] ERROR    Docker daemon doesn't seem to be responding. Please check it is up and running correctly before re-running the command.
 ```
 
+### MacOS after Docker Desktop upgrade
+
 We've seen this happen after a Docker Desktop upgrade. Defaults are changed and the Docker daemon no longer uses Unix sockets but TCP, or perhaps it does use Unix sockets but under a different path or user.
 
 What has worked for us in order to fix the issue is to make sure the following setting is enabled:
 ![docket-daemon-not-responding](/assets/images/screenshots/leverage-docket-desktop-daemon-issue.png "Docker daemon not responding")
 
 Note: that setting can be accessed by clicking on the Docker Desktop icon tray, and then clicking on "Settings...". Then click on the "Advanced" tab to find the checkbox.
 
+### Linux and Docker in Rootless mode
+
+The same problem might come from missing env variable `DOCKER_HOME`. `leverage` looks for Docker socket at `unix:///var/run/docker.sock` unless `DOCKER_HOME` is provided in environment. If you installed Docker in [Rootless mode](https://docs.docker.com/engine/security/rootless/), you need to remember to add `DOCKET_HOME` in you rc files
```

Comment:
```suggestion
The same problem might come from missing env variable `DOCKER_HOST`. `leverage` looks for Docker socket at `unix:///var/run/docker.sock` unless `DOCKER_HOST` is provided in environment. If you installed Docker in [Rootless mode](https://docs.docker.com/engine/security/rootless/), you need to remember to add `DOCKET_HOST` in you rc files
```
Should it rather be `DOCKER_HOST` like I suggest above?

Reply:
yes, you are right. There is one more typo there (DOCKET in the last `DOCKET_HOST`), therefore, I will just update this PR instead of committing your suggestion

Reply:
@rynkowsg  Awesome! Thanks for your contribution!